src/import/codegen.ts

```diff
@@ -384,14 +384,21 @@ export function generateHelmConstruct(typegen: TypeGenerator, def: HelmObjectDef
       code.close('};');
 
       code.line();
-      code.line('new Helm(scope, \'Helm\', finalProps)');
+      code.line('new Helm(scope, `Helm${id}`, finalProps)');
```

Comment:
Whoopsy, we missed passing the correct scope. 

```suggestion
      code.line('new Helm(this, 'Helm', finalProps)');
```